desc/grid.py

```diff
@@ -396,20 +396,22 @@ class Grid(_Grid):
     Parameters
     ----------
     nodes : ndarray of float, size(num_nodes,3)
-        node coordinates, in (rho,theta,zeta)
+        Node coordinates, in (rho,theta,zeta)
+    NFP : int
+        Number of field periods (Default = 1).
```

Comment:
What does having nfp!=1 do for custom grids?

Reply:
use eq.NFP as default instead?

Reply:
@f0uriest double check plotting and map coords that everything works with NFP=1

Reply:
looks like the only place we reference `Grid.NFP` is here: https://github.com/PlasmaControl/DESC/blob/da715883e689956c08bf8b03ffd0396d37fb4150/desc/plotting.py#L1660

Probably easiest to just change that to `eq.NFP` and leave `Grid` with NFP of 1